### ISSUE TITLE
Using tabindex to create a tab order that does not preserve meaning and operability

### ISSUE THREAD

**Issue by GGD4V** [MeD-DMC/Adapt-accessibilityfixes#96]
Issue: A focus order is being forced using tabindex="" with positive values. This prevent the natural focus order the page should have. For example, it skips the "Skip to main content" link and the top navigation, making them irrelevant. How to fix: Instead of using tabindex="" with positive values, make sure the "Skip to main content" link is the first element to get in focus by putting as the fist focus-able element in the DOM (which it already is?) and that all the other focus-able elements are in order in the DOM as well. If there's something you don't which to get in focus, you can use tabindex="-1" as long as it makes sense.

**Comment by GGD4V** [comment]
Confirmed skip on first tab in Safari. Should fix.

**Comment by Tamayuzu** [comment]
Code started, see comment //GitHub error 96

**Comment by Tamayuzu** [comment]
See https://github.com/MeD-DMC/Adapt-accessibilityfixes/pull/110
